src/docs/app-monitoring/user-defined-monitoring.md

```diff
@@ -212,9 +212,43 @@ For detail steps, please read the documents below:
 
 ## Monitor user-defined projects using Alertmanager
 
-Technical preview for our Openshift Clusters (OCP 4.10). It is possible to set up Alertmanager rule for a user-defined projects so that the granted user(s) by the monitoring-rules-edit role can create, modify, and deleting PrometheusRule custom resources for their project. It's coming with OCP 4.11.
+Alerting rules are created for user-defined projects and based on specific chosen metrics these alerts get triggered.
+Create a YAML file lets assume the name is app-alerting-rule.yaml and we can add rule named ```example-alert```, the rule triggers an alert when the ```version``` metric exposed by the sample service becomes ```0```.
```

Comment:
Inline code only needs a single backtick not three

Reply:
I don't think we need to be explicit about creating yaml files and applying them. If you look at other sections of the doc like **Expose the metrics from your app** we just provide the yaml without details on how to create the object. It's assumed that skill exists.